.github/workflows/bpf-checks.yaml

```diff
@@ -25,3 +25,28 @@ jobs:
       - uses: docker://cilium/coccicheck:1.0
         with:
           entrypoint: ./contrib/coccinelle/check-cocci.sh
+  build_all:
+    name: build datapath
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v2
+        with:
+          path: $HOME/.clang
+          key: llvm-10.0
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v1
+        with:
+          version: "10.0"
```

Comment:
In dev VM, we are using LLVM 11.0 as per https://github.com/cilium/packer-ci-build/pull/242. Maybe we can use 11.0 here as well, so that in case of failure, behavior will be consistent with local dev.

Reply:
We can't... yet. I will likely have to revert the LLVM version bump in the dev. VMs because it is causing verifier issues due to bytecode patterns LLVM generates.

Reply:
I'm not *so* worried about this since the code generation is not being tested, only C frontend handling. Worth keeping in mind if we leverage this build target for verifier testing in future though.